### PR TITLE
new confirmation page

### PR DESCRIPTION
**Pull request by mma1504**
**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [ ] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: packages/paste-website/src/pages/patterns/confirmations/index.mdx [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+        <Td> User wants to leave a flow but changes can’t be saved</Td>
+        <Td> **Stop [doing thing]?** <br/> Your changes will not be saved. <br/> Keep [doing thing] | Stop [doing thing] </Td>
+      </Tr>
+      <Tr verticalAlign="top">
+        <Td> The customer needs to swap regions to access the data.</Td>
+        <Td> **Use title to repeat action** <br/> Explain what could happen to the resource, and offer other options if available and appropriate.</Td>
+      </Tr>
+    </TBody>
+  </Table>
+</Box>
```
Comment (SiTaggart) [inline packages/paste-website/src/pages/patterns/confirmations/index.mdx:140]: Honestly, tables are much easier to do with pure Markdown syntax. https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables. It's very rare that we actually need to do Component pages tables

Comment (mma1504) [reply packages/paste-website/src/pages/patterns/confirmations/index.mdx]: Changed it!

**Review by nkrantz — APPROVED** [review]
🔥 🔥 🔥

**Review by SiTaggart — DISMISSED** [review]

File: packages/paste-website/src/pages/patterns/confirmations/index.mdx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        <Anchor href="/components/alert-dialog">Alert Dialog</Anchor>
+      </Heading>
+    </Card>
+  </Column>
+</Grid>
+
+```jsx
+// import all ingredients for the confirmations pattern
+
+import { AlertDialog } from *@twilio-paste/core/alert-dialog*;
```
[the remark this answers is not in the document]
Comment (nkrantz) [reply packages/paste-website/src/pages/patterns/confirmations/index.mdx]: fixed

**Review by serifluous — APPROVED** [review]

**Review by SiTaggart — APPROVED** [review]

**Pull request merged**